ts/components/LightboxGallery.md

```diff
@@ -1,16 +1,34 @@
 ```js
 const noop = () => {};
 
-const items = [
-  { objectURL: 'https://placekitten.com/800/600', contentType: 'image/jpeg' },
-  { objectURL: 'https://placekitten.com/900/600', contentType: 'image/jpeg' },
-  { objectURL: 'https://placekitten.com/980/800', contentType: 'image/jpeg' },
-  { objectURL: 'https://placekitten.com/656/540', contentType: 'image/jpeg' },
-  { objectURL: 'https://placekitten.com/762/400', contentType: 'image/jpeg' },
-  { objectURL: 'https://placekitten.com/920/620', contentType: 'image/jpeg' },
+const messages = [
```

Comment:
Made this change in my receive contact PR as well...

Reply:
Oh no, sorry, you mentioned that — totally forgot 😐